backend/grant/proposal/views.py

```diff
@@ -516,8 +516,9 @@ def post_contribution_confirmation(contribution_id, to, amount, txid):
         id=contribution_id).first()
 
     if not contribution:
-        # TODO: Log in sentry
-        print(f'Unknown contribution {contribution_id} confirmed with txid {txid}')
+        msg = f'Unknown contribution {contribution_id} confirmed with txid {txid}, amount {amount}'
+        capture_message(msg)
+        print(msg)
```

Comment:
This will likely conflict with #345. I made it a warn, but am not logging it to sentry. I like that you're doing that though, so we should definitely keep the capture_message.